### PR TITLE
Updating vm size for jobs. Placing e2e tests on deployed clusters.

### PR DESCRIPTION
**Pull request by kwoodson**
Updating instance sizes for the CI jobs.  Also reusing the cluster builds to run e2e tests on the desired clusters.

### CODE REVIEW

**Comment by kwoodson** [comment]
This will relies on https://github.com/openshift/openshift-ansible/pull/8917

File: sjb/config/test_cases/azure_build_base_image_centos.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ actions:
         -e "openshift_azure_input_image_prefix=centos7-root" \
         -e "openshift_azure_output_image_ns=images" \
         -e "openshift_azure_output_image_name=centos7-base-$(TZ=Etc/UTC date +%Y%m%d%H%M)" \
+        -e "openshift_azure_vm_size=Standard_D2s_v3" \
```
Comment (jim-minter) [inline sjb/config/test_cases/azure_build_base_image_centos.yml]: make this a job variable for consistency?

File: sjb/config/common/test_cases/origin_release_install_azure.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@ extensions:
       default_value: "ci-pr-images/"
     - name: "DEBUG_MAIL"
       default_value: "aos-azure@redhat.com"
+    - name: "AZURE_INSTANCE_SIZE"
```
Comment (jim-minter) [inline sjb/config/common/test_cases/origin_release_install_azure.yml]: rename AZURE_VM_SIZE for consistency

**Comment by jim-minter** [comment]
2 minor changes then can merge

**Pull request merged**